dev/src/index.ts

```diff
@@ -1117,9 +1147,23 @@ export class Firestore {
         'All onSnapshot() listeners must be unsubscribed before terminating the client.'
       );
     }
+    if (this.pendingOperationsCount > 0) {
+      return Promise.reject(
+        'All BulkWriter operations must be completed before terminating the client.'
```

Comment:
Mention that the user should call and await `close()`.

Reply:
Updated message.